cranelift/filetests/filetests/vcode/aarch64/extend-op.clif

```diff
@@ -11,6 +11,7 @@ block0(v0: i8):
 
 ; check:  stp fp, lr, [sp, #-16]!
 ; nextln:  mov fp, sp
+; nextln:  sxtb x0, w0
```

Comment:
Hmm, this isn't quite right: we should be able to rely directly on the `sxtb` extend-mode below; the separate instruction is redundant.